docs/mvp_morevp_interaction.md

```diff
@@ -0,0 +1,90 @@
+MVP vs MoreVP
+==
+
+Since both MVP standard exit and MoreVP in-flight exit provide an exit opportunity, it is possible to effectively double-spending. To prevent such possiblity, and to preserve the rule that honest owner can ignore IFE provided that he enjoys data availability, following scenarios are evaluated.
+
+#### Glossary
+
+* `IFE` - in-flight exit, active unless stated otherwise;
+* `SE` - standard exit, active unless stated otherwise;
+* `SE on (in|out)put` - standard exit on utxo mentioned in in-flight exit as input or output;
+* `->` denotes order of events;
+* `=>` denotes proposed solution / action;
+* `contract:` solution is contained to change in the contract;
+* `game:` solution is an interactive game, possibly with new functions exposed by the contract.
+
+### Possible scenarios
+
+1. SE on input -> IFE  
+    => (IMPLEMENTED) contract: IFE challenges SE (~ +30k gas)  
+    NOTE: can do that even if I remembered standardExitId(hash(tx), oindex) on SE, because I am getting the input txs all over again in IFE
+
+2. SE on input, finalized -> IFE  
+    => (IMPLEMENTED) contract: IFE is "as if non-canonical" and can't be canonized; finalized input is marked as done (req. additional flag on IFE)
```

Comment:
"finalized input is marked as done" <- imprecise sentence

Reply:
done